pkgs/development/node-packages/default.nix

```diff
@@ -131,6 +131,14 @@ let
       '';
     };
 
+    tsun-bin = super.tsun.overrideAttrs (oldAttrs: {
```

Comment:
What do you mean by "library usage"? Actually one could just override `tsun` directly:

```suggestion
    tsun = super.tsun.overrideAttrs (oldAttrs: {
```

Reply:
To be honest, i didn't look at tsun itself before trying it, while reviewing. @Mic92 you are right. tsun is not intended to be used as library.

Reply:
It technically could, but I agree with you both, it is very unlikely. I will update the PR